README.md

```diff
@@ -33,9 +33,10 @@ plugins: {
 
 			// Panning directions. Remove the appropriate direction to disable
 			// Eg. 'y' would only allow panning in the y direction
-			// A function that is called as the user is panning can also be used:
-			//   mode: function({chart, direction}: {chart: ChartJS.Chart, direction: 'x' | 'y'}) {
-			//     return direction === 'y'
+			// A function that is called as the user is panning and returns the
+			// available directions can also be used:
+			//   mode: function({ chart }: { chart: ChartJS.Chart }) {
```

Comment:
The `{ chart }: { chart: ChartJS.Chart }` syntax is confusing to me and not consistent with the rest of the README. Can we make it just `{ chart }` instead? I think people will know it's a chart.js chart and `ChartJS` isn't really a defined namespace anyway